Assets/Scripts/RoadCurve.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[System.Serializable]
+public class RoadCurve
+{
+    public int segmentInsertIndex, numEnterSegments, numHoldSegments, numExitSegments;
+    public float curveIntensity, elevationShift;
+    //public int SegmentInsertIndex { get => segmentInsertIndex; set => segmentInsertIndex = value; }
+    //public int NumEnterSegments { get => numEnterSegments; set => numEnterSegments = value; }
+    //public int NumHoldSegments { get => numHoldSegments; set => numHoldSegments = value; }
+    //public int NumExitSegments { get => numExitSegments; set => numExitSegments = value; }
+    //public int CurveIntensity { get => curveIntensity; set => curveIntensity = value; }     // Positive value to curve to the right; negative to curve left
```

Comment:
Don't forget to revisit these comments to see if they are delete-worthy or not.
Comment clutter can become a big problem in software dev teams, and often times comments are the first thing to become really outdated (people tend to forget about them).